### DISCUSSION TITLE
Use with Lumen Framework

### DISCUSSION THREAD

**Discussion by yogesh-jgu** [rebing/graphql-laravel#902] [General]
Can we use graphql with lumen framework?

**Comment by illambo** [comment]
From version 8.x lumen is not supported, pls check [changelog](https://github.com/rebing/graphql-laravel/blob/master/CHANGELOG.md#2021-11-15-800).

**Comment by mfn** [comment]
See also https://github.com/rebing/graphql-laravel/pull/722 where I explained it a bit more and also garnered feedback a bit.

**Comment by mfn** [comment]
The TL;DR: I rather spend time making sure this library works better with Octane than Lumen.
